docs/guide/protocols/relativehybridtopology.rst

```diff
@@ -81,6 +81,34 @@ Each Protocol simulation Unit carries out the following steps:
 
 Note: three different types of multistate sampling (i.e. replica swapping between lambda states) methods can be chosen; HREX, SAMS, and independent (no lambda swaps attempted). By default the HREX approach is selected, this can be altered using ``simulation_settings.sampler_method`` (default: ``repex``).
 
+Analysis
```

Comment:
Since the analysis part will be very similar for different HREX protocols (AHFE, RBFE), would it make sense to move the analysis to a subsection of 'working with results'? As a user, that's where I would be looking for these details, I think. I feel like we had discussed this somewhere else, but I don't remember your reasons for putting it here.